docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+
+   class MyApp extends StatelessWidget {
+     const MyApp({super.key});
+
+     @override
+     Widget build(BuildContext context) {
+       return MaterialApp(
+         title: 'Smart Machine App',
+         theme: ThemeData(
+           colorScheme: ColorScheme.fromSeed(seedColor: Colors.purple),
+         ),
+         home: MyHomePage(),
+       );
+     }
+   }
+
+   class MyHomePage extends StatelessWidget {
+     @override
+     Widget build(BuildContext context) {
+       return Scaffold(
+         body: Center(
+           child: Column(
+             mainAxisAlignment: MainAxisAlignment.center,
+             children: [
+               Text('Smart Machine App'),
+               SizedBox(height: 16),
+               ElevatedButton(onPressed: null, child: Text('Login')),
+             ],
+           ),
+         ),
+       );
+     }
+   }
+
+
+   ```
+
+   If you chose a name other than `Smart Machine App`, edit lines 13 and 30 with your own app title.
+
+### Launch the app
+
+In this section of the tutorial we'll build the app and see what it looks like so far.
+
+1. Open <file>lib/main.dart</file>.
+   In the bottom right corner of VS Code, you'll find a button that shows the current target device.
+   Click the button to change your target device.
+   Make sure that you have your target device selected before you continue.
+
+   {{<imgproc src="/tutorials/flutter-app/target-device-button.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="A VS Code window with main.dart open. In the bottom right corner, there is a target device button labeled 'iPhone 14 (ios simulator)'.">}}
+
+2. With <file>lib/main.dart</file> still open, find the "Start Debugging" button in the upper right corner of the VS Code window.
+   Click it.
+
+   {{<imgproc src="/tutorials/flutter-app/main-play.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="A VS Code window with main.dart open. In the top right corner, there is a button labeled 'Start Debugging' with an icon of a bug and a 'play' symbol.">}}
+
+   A window should open up, displaying a rendering of your smart machine app:
+
+   {{<imgproc src="/tutorials/flutter-app/render1.png" resize="1200x" style="max-width:300px" declared-dimensions="true" alt="Rendering of an iPhone with the app open on the screen. The screen is white with the words 'Smart Machine App' in the middle, above a grayed-out login button.">}}
+
+## Add app navigation
+
+### Add a new screen
+
+Great work so far!
+You have your app running, with a single screen and an inactive button.
+Next you will add a new screen that pulls in some information from your {{< glossary_tooltip term_id="organization" text="organization" >}} in Viam, and then navigate to that screen from the login button.
+
+In the VS Code file explorer, right click <file>lib/</file> and click **New File** - let’s name this new file <file>home_screen.dart</file>.
+
+Paste the contents of this code block into that new file:
```

Comment:
Nit: mention of "code block" gently hints a VS Code or my_new_app UI element with matching text. Users more familiar with "the code below" or similar:

```suggestion
Paste the following code into the <file>home_screen.dart</file> file you just created:
```